modules/gdscript/tests/scripts/analyzer/features/enums_in_range_call.gd

```diff
@@ -0,0 +1,9 @@
+enum E { E0 = 0, E3 = 3 }
+
+func test():
+	var total := 0
+	for value in range(E.E0, E.E3):
+		var inferrable := value
```

Comment:
Codespell wants this spelled "inferable".

Reply:
Done.